extern/ttconv/pprdrv_tt.cpp

```diff
@@ -1055,7 +1055,9 @@ void ttfont_CharStrings(TTStreamWriter& stream, struct TTFONT *font, std::vector
     post_format = getFixed( font->post_table );
 
     /* Emmit the start of the PostScript code to define the dictionary. */
-    stream.printf("/CharStrings %d dict dup begin\n", glyph_ids.size());
+    stream.printf("/CharStrings %d dict dup begin\n", glyph_ids.size()+1);
+    /* PS Lang Ref says ChatStrings must contain an entry for .notdef */
```

Comment:
Yes, thanks for catching that and for the section / table suggestion.  I tried fixing/squashing/pushing.  I haven't done many PRs, so if I made a mistake and you can fix it please do.  Thanks for matplotlib.  I use it and pandas at work instead of IDL and it makes my work easier to do